### ISSUE TITLE
SET a dict

### ISSUE THREAD

**Issue by g3rd** [emehrkay/Pypher#5]
Hello,

First off, thanks for this project. I started to write one today. Then stumbled upon this, and stopped mine.

How would I write:

```
CREATE (u:User) SET u = {props};
```

or

```
MATCH (u:User {username: {username}}) SET u += {props};
```

Thanks again!

**Comment by emehrkay** [comment]
Great catch. This was something that I was meaning to implement, but has completely feel off of my radar.  

It would be pretty easy to add support for setting a dict in python and returning the curly brace-structure in cypher. However, I remember seeing an example that looked something like `RETURN {.name}` and I not too sure how to handle it.

If we start with something like:

```python
p.SET.u += {'name': 'some name', 'age' 999}
```

What does that look like in cypher? Is it `SET u += {'name': $name_var, 'age': $age_var}` or something different? And what are the other permutations of that dict-like structure in cypher that we'd need to account for in python? 

It isn't too difficult to add, I just want to make sure we get it right. I will check the documentation for more examples. Please drop some links if you have any.

Thanks

**Comment by emehrkay** [comment]
Addressed with version 0.7.0
